docs/web/key-concepts.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Key Concepts
-description: Geocortex Web - Learn about key concepts of the Geocortex Web Framework
+description: Geocortex Web - Learn about key concepts of Geocortex Web
 hide_table_of_contents: true
 ---
 
```

Comment:
Do you maybe want to try changing the h4 headings to h3? I notice there's no TOC for this page.

Reply:
I originally got rid of the TOC with the h4 headings because I didn't feel a TOC helped present the content? this page is really only one topic, so I thought it was best to just omit the TOC

Reply:
Changed the headings to h3 but still left out TOC for reason stated above.